sflvault/config/config.py

```diff
@@ -0,0 +1,36 @@
+import os
+import yaml
+
+
+class Config():
+
+    def __init__(self, path):
+
+        self.default_config_location = path
+
+        try:
+            with open(self.default_config_location, 'r') as f:
+                self.config = yaml.safe_load(f)
+        except IOError:
+            os.sys.exit("Error while reading configuration file")
+
+    def get_config(self):
+        return self.config
+
+    def get_config_key_value(self, key):
+        try:
+            return self.config[key]
+        except KeyError:
+            print("Error while fetching the key %s from configuration".format(key))
```

Comment:
These `prints` should also be `raises` instead. Perhaps with a special `ConfigException` :)